app/Http/Controllers/Internlink/InternlinkController.php

```diff
@@ -212,12 +209,8 @@ public function updateContact(Request $request)
             return redirect()->back()->withErrors($form->getErrors())->withInput()->with('alert-warning', 'Error in form input!');
         }
 
-        $link = Link::where('member_id', $member->id)->get()->first();
-
+        $link = $member->internlink;
         $link->resetContact();
```

Comment:
I think that the entire function could be removed since it's now being populated automatically?